test/SecTester.Bus.Tests/Internal/JsonHttpMethodEnumerationStringConverterTests.cs

```diff
@@ -0,0 +1,59 @@
+namespace SecTester.Bus.Tests.Internal;
+
+public class JsonHttpMethodEnumerationStringConverterTests
+{
+  private readonly DefaultMessageSerializer _sut = new();
+
+  public static readonly IEnumerable<object[]> HttpMethodEnumerable = new List<object[]>()
+  {
+    new object[] { HttpMethod.Delete, @"""DELETE""", typeof(HttpMethod) },
```

Comment:
The type is redundant for all cases, you can pass it within `Deserialize_GivenString_ReturnEnumValue`
```suggestion
    new object[] { HttpMethod.Delete, @"""DELETE""" },
```